tests/50federation/40devicelists.pl

```diff
@@ -588,9 +604,13 @@
          )
       })->then( sub {
          ( $first_keys_query_body ) = @_;
+         log_if_fail("first_keys_query_body", $first_keys_query_body);
```

Comment:
```suggestion
         log_if_fail "first_keys_query_body", $first_keys_query_body;
```

Reply:
Erm is that correct? I thought a comma outside of a brackets was a C-style comma operator. (i.e. `a,b` will evaluate `a`, discard the result, then evaluate and return `b`.

Reply:
It's what we do elsewhere :shrug: